.github/workflows/kubevious-manifests-ci.yaml

```diff
@@ -0,0 +1,62 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: kubevious-manifests-ci
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - 'helm-chart/**'
+      - 'kustomize/**'
+      - '.github/workflows/kubevious-manifests-ci.yaml'
+  pull_request:
+    paths:
+      - 'helm-chart/**'
+      - 'kustomize/**'
+      - '.github/workflows/kubevious-manifests-ci.yaml'
+permissions:
+  contents: read
+jobs:
+  kubevious-manifests-ci:
+    runs-on: ubuntu-22.04
+    timeout-minutes: 1
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Validate kubernetes-manifests
+        id: kubernetes-manifests-validation
+        uses: kubevious/cli@v1.0.58
+        with:
+          manifests: kubernetes-manifests
+          skip_rules: container-latest-image
+
+      - name: Validate helm-chart
+        id: helm-chart-validation
+        uses: kubevious/cli@v1.0.58
+        with:
+          manifests: helm-chart
+
+      - name: Validate istio-manifests
+        id: istio-manifests-validation
+        uses: kubevious/cli@v1.0.58
+        with:
+          manifests: istio-manifests https://github.com/istio/istio/raw/master/manifests/charts/base/crds/crd-all.gen.yaml
+
+      - name: Validate kustomize
+        id: kustomize-validation
+        uses: kubevious/cli@v1.0.58
```

Comment:
I've pinned `kubevious/cli` to a specific version mainly for security reasons, but also in case `main` introduces breaking changes. Hope this is okay. :)